src/Hamiltonians/ExtendedHubbardReal1D.jl

```diff
@@ -1,12 +1,12 @@
 """
-    ExtendedHubbardReal1D(address; u=1.0, v=1.0, t=1.0, boundary_condition=:periodic)
+    ExtendedHubbardReal1D(address; u=1.0, v=1.0, t=1.0, boundary_condition=:periodic, power=:nearest_neighbour)
```

Comment:
```suggestion
    ExtendedHubbardReal1D(address; u=1.0, v=1.0, t=1.0, boundary_condition=:periodic, power=nothing)
```
This was not consistent with the code or the rest of the documentation.